<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceDB</class>
 <widget class="QDialog" name="ChoiceDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Data base</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/fileopen.png</normalon>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>247</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="newButton">
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/image/filenew.png</normalon>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="skipThisWindow">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Skip this window on start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="delButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>List of DB:</string>
     </property>
     <property name="buddy">
      <cstring>listOfBDs</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QListWidget" name="listOfBDs"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChoiceDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
